app/components/phone_input_component.html.erb

```diff
@@ -32,10 +32,9 @@
         :phone,
         class: 'usa-label',
       ) { t('two_factor_authentication.phone_label') } %>
-  <div class="margin-bottom-1 usa-hint js">
-    <%= t('forms.example') %>
-    <span class="phone-input__example"></span>
-  </div>
+  <%= f.hint safe_join(
+        [t('forms.example'), ' ', content_tag('span', '', class: 'phone-input__example')],
+      ) %>
```

Comment:
what if we did a `<span>` for the body of the hint instead of having to safe_join?

```suggestion
  <%= f.hint(content_tag(:span) do
    <%= t('forms.example') %>
    <span class="phone-input__example"></span>
  end) %>
```

Reply:
Good thought. I made a few adjustments, but implemented in 90c726261:

- To avoid the redundant wrapper, I opted for [`capture`](https://api.rubyonrails.org/v6.1.4/classes/ActionView/Helpers/CaptureHelper.html#method-i-capture) instead of `content_tag`
- Needs an extra set of `%>` and `<%` to allow for markup / ERB output in the block.

Reply:
ooo TIL capture, thank you